R/frosting.R

```diff
@@ -269,18 +269,6 @@ apply_frosting.epi_workflow <-
   }
 
 #' @export
```

Comment:
We still want a print method for frosting (it can just call `print_frosting()`). That way if you have a frosting object, it won't just print the unformatted list.